files/build_templates/multi_instance/bgp@.service.j2

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=BGP container
+#Requires=updategraph.service
```

Comment:
@SuvarnaMeenakshi @lguohan : we don't need this dependency for Multi Instance ? Single Instance still have this dependency.

Reply:
support for updategraph and config-setup service for multi-asic platform will be added in a separate PR. As updategraph service is not updated to support multiple config_db, this dependency cannot be added here.